tests/test_linearGaussian_apt.py

```diff
@@ -98,8 +98,6 @@ def test_apt_on_linearGaussian_based_on_mmd(
     ), f"MMD={mmd} is more than 2 stds above the average performance."
 
 
-test_apt_on_linearGaussian_based_on_mmd(3, "gaussian", "snpe_c", 10)
```

Comment:
This seems like an unrelated change to this commit, isn't it?

Reply:
Yes, it was a left-over from some other commit